Azure/sonic-buildimage
# [device] as4630_54pe add custom_led.bin file

Signed-off-by: derek_sun derek_sun@edge-core.com

- What I did
Change custom_led.bin binary files loading rule.

- How I did it
1) Add custom_led.bin file.
2) Modify led_proc_init.soc for loading LED command.

- How to verify it
1) Running SONiC to verify port link-up and front port LED status.

- Description for the changelog
Add custom_led.bin and modify led_proc_init.soc to loading custom_led.bin file.

- A picture of a cute animal (not mandatory but encouraged)
